src/pages/register/index.jsx

```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { 
+    Paper, 
+    Grid, 
+    Container, 
+    Avatar, 
+    InputLabel, 
+    OutlinedInput, 
+    InputAdornment, 
+    IconButton, 
+    Button, 
+    Link  
+} from "@material-ui/core";
+import { Visibility } from "@mui/icons-material";
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import useStyles from './register.styles';
+import logoLogin from '../../assets/logo/logoregister.png';
+import { useState } from 'react';
+
+const Login = () => {
+    const classes = useStyles();
+    const [email, setEmail] = useState();
+    const url='http://3.23.86.147/server/autenthication/register';
+    const [values, setValues] = React.useState({
+        password: '',
+        repassword: '',
+        showPassword: false,
+        showRePassword: false,
+    });
+    const passw = values.password;
+    const repassw = values.repassword;
+
+    async function loginUser(credentials) {
+        return fetch(url, {
+          method: 'POST',
```

Comment:
Don't use the service in the component put te request into async folder use https://github.com/Ditmar/plagiarist-group5/blob/master/src/async/api.js and create a service such as  https://github.com/Ditmar/plagiarist-group5/blob/master/src/async/services/socketIO.js